src/peer_proxy_negotiate_auth.cc

```diff
@@ -307,13 +306,7 @@ int krb5_create_cache(char *kf, char *pn) {
                    error_message(code));
             return (1);
         }
-#elif HAVE_LIBHEIMDAL_KRB5 && HAVE_KRB5_GET_MAX_TIME_SKEW
         skew = krb5_get_max_time_skew(kparam.context);
```

Comment:
In my question below, the phrase "profile condition" stands for `HAVE_PROFILE_H && HAVE_KRB5_GET_PROFILE && HAVE_PROFILE_GET_INTEGER && HAVE_PROFILE_RELEASE` condition a few lines above this code. I also assume, like this PR does, that we have access to krb5_get_max_time_skew().

In official code, this line is compiled only when profile condition is false.

In PR code, this line is compiled only when profile condition is true.

If this change is not intentional, please fix.

Reply:
fixed in 7cce5b64e5